src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/SyntaxTokenExtensions.cs

```diff
@@ -36,19 +36,8 @@ public static IEnumerable<SyntaxNode> GetAncestors(this SyntaxToken token, Func<
             : [];
     }
 
-    public static SyntaxNode? GetCommonRoot(this SyntaxToken token1, SyntaxToken token2)
-    {
-        Contract.ThrowIfTrue(token1.RawKind == 0 || token2.RawKind == 0);
-
-        // find common starting node from two tokens.
-        // as long as two tokens belong to same tree, there must be at least on common root (Ex, compilation unit)
-        if (token1.Parent == null || token2.Parent == null)
-        {
-            return null;
-        }
```

Comment:
these can't happen in real code.  so just changed to GetrequiredParent() to ensure that.